xml/libvirt_networking.xml

```diff
@@ -8,958 +8,957 @@
  <title>Managing Networks</title>
  <info>
   <abstract>
-   <para>
-    This chapter introduces common networking configurations supported by
-    &libvirt;. It does not depend on the hypervisor used. It is valid for all
-    hypervisors supported by &libvirt;, such as &kvm; or &xen;. These setups
-    can be achieved using both the graphical interface of &vmm; and the command
-    line tool <command>virsh</command>.
+    <para>
+     This chapter describes common network configurations for a &vmhost;,
+     including those supported natively by the &vmhost; and &libvirt;.
+     The configurations are valid for all hypervisors supported by &productname;,
+     such as &kvm; or &xen;.
    </para>
   </abstract>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:bugtracker></dm:bugtracker>
   </dm:docmanager>
  </info>
  <para>
-  There are two common network setups to provide a &vmguest; with a network
-  connection:
+  There are two common network configurations to provide a &vmguest; with a
+  network connection:
  </para>
  <itemizedlist>
   <listitem>
    <para>
-    A <emphasis>virtual network</emphasis> for the guest
+    A <emphasis>network bridge</emphasis> acting as a Layer 2 switch
    </para>
   </listitem>
   <listitem>
    <para>
-    A <emphasis>network bridge</emphasis> over a host's physical network
-    interface that the guest can use
+    A <emphasis>virtual network</emphasis> managed by &libvirt; with Layer 3
+    forwarding enabled
```

Comment:
Virtual network: libvirt manages the network, relying on layer 3 forwarding

this configuration is best suited for YYY

Configuring this kind of networking for guest VMs means that VMs will be able to (blabla) and hosts located in network ZZZ will be able to reach the VM.

Consider these consequences of choosing this networking type for your guest VMs (QQQ notable consequences and side effects)